torchgeo/datasets/forestdamage.py

```diff
@@ -40,7 +40,10 @@ def parse_pascal_voc(path: str) -> Dict[str, Any]:
             int(bndbox.find("xmax").text),  # type: ignore[union-attr, arg-type]
             int(bndbox.find("ymax").text),  # type: ignore[union-attr, arg-type]
         ]
-        label = obj.find("damage").text  # type: ignore[union-attr]
+        if obj.find("damage") is not None:
+            label = obj.find("damage").text  # type: ignore[union-attr]
```

Comment:
Any idea why the type ignore is still required? I believe `obj.find("damage")` is of type `Union[Element, None]` which is why the ignore was previously required. If we explicitly check for None, I would think this would no longer be needed.

Also see the hoops I had to jump through to get this working for FAIR1M by casting everything to the correct type.

Reply:
mypy says that `obj.find("damage")` is `error: Item "None" of "Optional[Element]" has no attribute "text"  [union-attr]`.

Reply:
Ah, I think I understand the problem. Mypy itself is smart enough to figure this out. The following file type checks properly even though "None" has no attribute "replace":
```python
from typing import Union

x: Union[str, None] = 'foo'

if x is not None:
    print(x.replace('foo', 'bar'))
```
The problem here is that we are running `obj.find("damage")` twice and mypy has no way of knowing that the output will be of the same type both times. Also, this is slower than it needs to be. Can you store the output of `obj.find("damage")` in a variable and then use that variable instead? This will speed things up and avoid the type ignore.

Reply:
awesome, thanks!